python.pri

```diff
@@ -29,18 +29,17 @@
 # variable to specify an absolute path to a Python binary to use.
 # Setting this environment variable overrides any querying.
 
-win32 {
-	PYTHON = $$(MUMBLE_PYTHON)
-	isEmpty(PYTHON) {
+PYTHON = $$(MUMBLE_PYTHON)
+
+# If the MUMBLE_PYTHON environment variable isn't set
+isEmpty(PYTHON) {
+	# If we're on Qt 4 on win32, or we're building from Windows, use "python"
+	equal(QMAKE_HOST.os, "Windows")|(equal(QMAKE_VERSION_INT_MAJOR, 4):win32) {
```

Comment:
Probably needs a qt.pri include to use the version string.

Also not sure og this even runs. Did you test it?